lib/NodeStatus.js

```diff
@@ -79,28 +79,40 @@ NodeStatus.childChanged = function(node, childStatus, childArgIndex=null) {
     newNode.content = childStatus.newNode;
     oldNode.content = childStatus.oldNode;
     substeps.map(step => {
-      let newStep = clone(node, false);
-      newStep.content = step;
-      return newStep;
+      let newNode = clone(node, false);
```

Comment:
nit: put old first? I wasn't consistent with it before but it'd be nice to do that